src/main/pages/che-7/end-user-guide/proc_writing-a-meta-yaml-file-and-adding-it-to-a-plug-in-registry.adoc

```diff
@@ -24,11 +23,79 @@ description:                                                     <8>
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg     <9>
 repository:                                                     <10>
 category:                                                       <11>
+firstPublicationDate:                                           <12>
+latestUpdateDate:                                               <13>
+deprecate:                                                      <14>
+    autoMigrate:                                                <15>
```

Comment:
If I am not mistaken, it was implemented as 
```
deprecate {
    "automigrate":  true,
    "migrateTo": "another_publisher/another_plugin_id:1.0.0"
},
```
in this PR - https://github.com/eclipse/che-plugin-registry/pull/228

https://github.com/eclipse/che-plugin-registry/pull/228/files#diff-04c6e90faac2675aa89e2176d2eec7d8R186

Reply:
Than we have a bug in the spec description: https://github.com/eclipse/che-plugin-registry/blame/2b5d8a85c0d1aaf23f8a1939625a44c26c9ede55/README.md#L92

Reply:
Yes. Exactly. It is a new issue